services/business/checkers/virtual_services/single_host_checker_test.go

```diff
@@ -176,6 +258,26 @@ func buildVirtualService(name, host string) kubernetes.IstioObject {
 	return vs
 }
 
+func buildVirtualServiceWithGateway(name, host, gateway string) kubernetes.IstioObject {
+	vs := (&kubernetes.VirtualService{
```

Comment:
We should avoid creating these new direct objects, given that there's code to create this testdata.

Such as (something like this, difficult in the github UI): ``data.AddGatewayToVirtualService(.., data.CreateEmptyVirtualService(name, "bookinfo", []string{host}))``